### PR TITLE
fix(wallet): fix get addresses offset and limit

### PR DESCRIPTION
**Pull request by mariocao**

### CODE REVIEW

File: wallet/src/actors/app/handlers/get_addresses.rs [hunk 1]
```diff
@@ -27,11 +25,9 @@ impl Handler<GetAddressesRequest> for app::App {
         let offset = msg
             .offset
             .unwrap_or_else(|| constants::DEFAULT_PAGINATION_OFFSET);
-        let limit = cmp::min(
-            msg.offset
-                .unwrap_or_else(|| constants::DEFAULT_PAGINATION_LIMIT),
-            constants::MAX_PAGINATION_LIMIT,
```
Comment (tmpolaczyk) [inline wallet/src/actors/app/handlers/get_addresses.rs:33]: Why remove the limit?

File: wallet/src/actors/app/handlers/get_addresses.rs [hunk 2]
```diff
@@ -28,7 +28,7 @@ impl Handler<GetAddressesRequest> for app::App {
             .offset
             .unwrap_or_else(|| constants::DEFAULT_PAGINATION_OFFSET);
         let limit = cmp::min(
-            msg.offset
+            msg.limit
                 .unwrap_or_else(|| constants::DEFAULT_PAGINATION_LIMIT),
             constants::MAX_PAGINATION_LIMIT,
```
Comment (tmpolaczyk) [inline wallet/src/actors/app/handlers/get_addresses.rs:33]: Why keep the limit? :smiley:

Comment (mariocao) [reply wallet/src/actors/app/handlers/get_addresses.rs:33]: To use a similar logic as in `get_transactions` :laughing:

Comment (mariocao) [reply wallet/src/actors/app/handlers/get_addresses.rs]: I removed it along with the one in `get_transactions`

**Pull request merged**